lib/scaffolds/tab.dart

```diff
@@ -34,9 +34,11 @@ class TabScaffold extends StatelessWidget {
             onValueChanged: onTabSwitch,
             children: tabs.asMap().map((key, text) => MapEntry(
                 key,
-                Padding(
-                  padding: const EdgeInsets.symmetric(horizontal: 8),
-                  child: Text(text),
+                Container(
+                  width: 100,
+                  child: Center(
+                    child: Text(text),
+                  ),
                 ))),
```

Comment:
How about adding `Row > Expanded` as `CupertinoSlidingSegmentedControl`'s parent? Then it would expand to the full width.

Reply:
The magic number 100 would break again if the word becomes long enough at next time.

Reply:
I tried using `Row > Expanded` as I mentioned in the issue. Didn't work

Reply:
I'm not sure if I made it clear. The following code seems work fine.

```dart

  Widget _buildTitle(BuildContext context) {
    switch (Provider.of<ThemeModel>(context).theme) {
      case AppThemeType.cupertino:
        return DefaultTextStyle(
          style: DefaultTextStyle.of(context).style.copyWith(fontSize: 14),
          child: Row(
            children: [
              Expanded(
                child: CupertinoSlidingSegmentedControl(
                  groupValue: activeTab,
                  onValueChanged: onTabSwitch,
                  children: tabs.asMap().map((key, text) => MapEntry(
                      key,
                      Padding(
                        padding: const EdgeInsets.symmetric(horizontal: 8),
                        child: Text(text),
                      ))),
                ),
              ),
            ],
          ),
        );
      default:
        return title;
    }
  }
```

![image](https://user-images.githubusercontent.com/9524411/94985019-0e155380-0585-11eb-9d27-51afb22f3a60.png)